src/chapters/thymeleaf-views/studio.rst

```diff
@@ -0,0 +1,71 @@
+Studio: Spa Day!
```

Comment:
Sally, this is a clever idea and a nice way to avoid a bland studio theme!

Reply:
Thank you!